cilium/cilium.io
# fix: pages design

**This PR:**
- Fixes Brand page design issues — [Problem with description](https://github.com/cilium/cilium.io/assets/105175118/2d97b457-ef93-415f-86ae-548d9e87bdc1);
- Fixes Enterprise page design issues — [Problem with mobile adaptation](https://github.com/cilium/cilium.io/assets/105175118/664f81c6-0d91-4487-b4d9-fe7e7109df44);
- Refactor all Case studies pages;


**Steps to test**

- Open the [Enterprise page](https://deploy-preview-320--cilium.netlify.app/enterprise/)
- Open the [Brand page](https://deploy-preview-320--cilium.netlify.app/brand/)
- Open the Case studies pages from [Get started page](https://deploy-preview-320--cilium.netlify.app/get-started/)
- Confirm that everything looks and works as expected
